src/libinjection_xss.c

```diff
@@ -135,6 +135,97 @@ static int html_decode_char_at(const char* src, size_t len, size_t* consumed)
     }
 }
 
+/*
+ * view-source:
+ * data:
+ * javascript:
+ * events:
+ */
+static stringtype_t BLACKATTREVENT[] = {
```

Comment:
Not sure we want to address this here, but this is lacking quite a few event handlers.
Also, since they all have the `on` prefix we can, for example, store just `abort` instead of `onabort` and simplify the matching done afterwards.

Reply:
@victorhora Here https://github.com/client9/libinjection/pull/153#issuecomment-760255710 this user @rcanavan made an addition of this very patch. The user seems to be confused with the existence of the multiple repositories for libinjection, yet, he posted valid information; There, he mentioned this list - https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers

@pmjdebruijn  @victorhora @rcanavan I am wondering if you guys can work together on this.

@migueldemoura this external list seems to be more "complete". I will extract a list from google chrome source code, just for the sake of comparison.

Reply:
The spec and MDN don't contain quite a few non-standard events, so it's best to pull them from the browser engine's sources as you mentioned:

https://source.chromium.org/chromium/chromium/src/+/master:out/Debug/gen/third_party/blink/renderer/core/event_type_names.cc
https://hg.mozilla.org/mozilla-central/file/tip/dom/events/EventNameList.h 
https://trac.webkit.org/browser/webkit/trunk/Source/WebCore/dom/EventNames.h

There's obviously also events that are specific to certain JavaScript frameworks, but that's probably out-of-scope for this.